src/pages/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -0,0 +1,83 @@
+import { ArrowBackIcon, ChevronRightIcon, CopyIcon } from '@chakra-ui/icons';
+import {
+  Box,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Text,
+  Tooltip,
+  useBreakpointValue,
+  useToast,
+} from '@chakra-ui/react';
+import { useRouter } from 'next/router';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface DashboardHeaderProps {
+  pageTitle: string;
+}
+
+const DashboardHeader: React.FC<DashboardHeaderProps> = function Dashboard({ pageTitle }) {
```

Comment:
nit: we don't use the `function` keyword for any of the React components or methods